modules/git/command.go

```diff
@@ -227,6 +227,12 @@ func (c *Command) RunInDirWithEnv(dir string, env []string) (string, error) {
 	return string(stdout), nil
 }
 
+// RunInDirWithEnvBytes executes the command in given directory
+// and returns stdout in []byte and error (combined with stderr).
+func (c *Command) RunInDirWithEnvBytes(dir string, env []string) ([]byte, error) {
```

Comment:
The wrap function seems unnecessary?

Reply:
Honestly, this file scares me lol. They all look like mostly unnecessary wrapper functions to me. But I was trying to go along with the style. 🤷‍♂️

Reply:
Just use `RunInDirTimeoutEnv(env, -1, dir)` directly instead. At some point we'll hopefully destroy all of these and replace them with a single RunOptions{} that can have things set. 

Honestly, usually what happens is I end up escalating all the way up to FullPipeline to use Readers because a lot of functions misuse storing unbounded results and/or the error munging with these downstream functions ends up being unhelpful.